librz/type/format.c

```diff
@@ -582,7 +582,7 @@ static void rz_type_format_time(RzStrBuf *outbuf, int endian, int mode,
 // TODO: support unsigned int?
 static void rz_type_format_hex(RzStrBuf *outbuf, int endian, int mode,
 	const char *setval, ut64 seeki, ut8 *buf, int i, int size) {
-	ut64 addr;
+	ut64 addr = 0xadc0ffeeadc0ffee;
```

Comment:
just use 0 ?

Reply:
0 is a very common value but ok d0868ab53fd09f22e2214677034e62d0a63f947e